client/components/import/import.js

```diff
@@ -11,48 +11,105 @@ const ImportPopup = BlazeComponent.extendComponent({
     return 'importPopup';
   },
 
-  events() {
-    return [{
-      'submit': (evt) => {
-        evt.preventDefault();
-        const dataJson = $(evt.currentTarget).find('.js-import-json').val();
-        let dataObject;
-        try {
-          dataObject = JSON.parse(dataJson);
-        } catch (e) {
-          this.setError('error-json-malformed');
-          return;
-        }
-        Meteor.call(this.getMethodName(), dataObject, this.getAdditionalData(),
-          (error, response) => {
-            if (error) {
-              this.setError(error.error);
-            } else {
-              Filter.addException(response);
-              this.onFinish(response);
-            }
-          }
-        );
-      },
-    }];
+  jsonText() {
+    return Session.get('import.text');
```

Comment:
Since Session is a global store I totally like your namespacing convention, I will use it in the future — until eventually we have better build-in solution. :+1: